go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/golangci/golangci-lint v1.58.2
 	github.com/hashicorp/terraform-plugin-docs v0.19.4
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.34.0
-	github.com/hewlettpackard/hpegl-metal-client v1.5.22
+	github.com/hewlettpackard/hpegl-metal-client v1.5.23-0.20240923072040-5b07002ffb48
```

Comment:
https://github.com/HewlettPackard/hpegl-metal-client/pull/132 for client change.
Will update the actual tag post merging of this pr.

Reply:
Updated the pr with released client-https://github.com/HewlettPackard/hpegl-metal-client/releases/tag/v1.5.23